src/openms/include/OpenMS/QC/TIC.h

```diff
@@ -71,7 +70,16 @@ namespace OpenMS
     @param bin_size RT bin size in seconds
     @return TIC Chromatogram
     **/
-    void compute(const MSExperiment &exp, float bin_size=0);
+    struct Result
+    {
+      std::vector<UInt> intensities;
+      std::vector<float> retention_times;
+      UInt area = 0;
```

Comment:
can you document what these members indicate?

Reply:
added comments in code and updated description for compute function